Packs/Legacy/Integrations/EWSv2/EWSv2.py

```diff
@@ -946,6 +946,19 @@ def str_to_camel_case(snake_str):
     return str_to_camel_case(value)
 
 
+def email_ec(item):
+    return {
+        'CC': [] if not item.cc_recipients else [mailbox.email_address for mailbox in item.cc_recipients],
+        'BCC': [] if not item.bcc_recipients else [mailbox.email_address for mailbox in item.bcc_recipients],
+        'To': [] if not item.to_recipients else [mailbox.email_address for mailbox in item.to_recipients],
+        'From': item.author.email_address,
+        'Subject': item.subject,
+        'Text': item.text_body,
+        'HTML': item.body,
+        'HeadersMap': {header.name: header.value for header in item.headers}
```

Comment:
```suggestion
        'HeadersMap': {header.name: header.value for header in item.headers},
```

Reply:
Why we need to add comma in last item of dict or item?